### PR TITLE
fix(Button): Implementing Contrast Theme and Updating Dark Theme

### PR DESCRIPTION
**Pull request by bcalvery**
# Adding Contrast Theme styles for Button

Contrast:
image
image

Dark:
image
image

### CODE REVIEW

**Review by alinais — APPROVED** [review]

**Review by mnajdova — APPROVED** [review]

**Review by kuzhelov — APPROVED** [review]
👍

File: src/themes/teams/components/Button/buttonStyles.ts [hunk 1]
```diff
@@ -216,34 +256,7 @@ const buttonStyles: ComponentSlotStylesInput<ButtonProps & ButtonState, any> = {
...
-        padding: 0,
-        borderRadius: circularRadius,
-      }),
-
-      ...(fluid && {
-        width: '100%',
-        maxWidth: '100%',
-      }),
-
+      // Overrides for "disabled" buttons
```
Comment (levithomason) [inline src/themes/teams/components/Button/buttonStyles.ts:258]: nit: next line "says" the same thing.

**Review by levithomason — APPROVED** [review]
see one nit

File: src/themes/teams/components/Button/buttonStyles.ts (hunk 1 above)
Comment (alinais) [reply src/themes/teams/components/Button/buttonStyles.ts:258]: We have these comments in all the file, not only here. We can consider it as part of some cleanup task.

**Pull request merged**